contracts/verifying/VerifyingSingletonPaymasterV2.sol

```diff
@@ -1,9 +1,363 @@
-// SPDX-License-Identifier: MIT
-pragma solidity ^0.8.17;
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.8.17;
```

Comment:
All out verifying paymasters have been : // SPDX-License-Identifier: GPL-3.0
since it's inspired from (and a lot of code same) infinitism's verifying paymaster sample. 
so if something we use is GPL 3 ours also should be GPL 3
we have made some changes to make it singleton that's our own logic.

please don't be confused as license being changed. as it was intermediately draft contract before

Reply:
Do we need to sync on SCW to ensure we're not breaking any licenses by using code from various sources?

Reply:
they are separate contracts separate purposes.

Reply:
I had this doubt though when we did c4 audit and we chose to keep MIT for every contract. If needed we can change this later